source/gameengine/Rasterizer/RAS_OpenGLRasterizer/RAS_OpenGLRasterizer.cpp

```diff
@@ -932,6 +932,29 @@ void RAS_OpenGLRasterizer::DrawDebugCameraFrustum(SCA_IScene *scene, const MT_Ma
 		MT_Vector4(0.8f, 0.5f, 0.0f, 1.0f));
 }
 
+void RAS_OpenGLRasterizer::DrawDebugLightFrustum(SCA_IScene *scene, const MT_Matrix4x4& projmat, const MT_Matrix4x4& viewmat)
```

Comment:
Both functions could eventually be merged with a switch ob->type == OB_LAMP or OB_CAMERA or something like that but for now DrawDebugCameraFrustum use const MT_Matrix4x4 mv = (projmat * viewmat).inverse();
and DrawDebugLightFrustum use const MT_Matrix4x4 mv = (projmat * viewmat).inverse();